### PR TITLE
feat(measurement): added spatial axes schema

### PR DESCRIPTION
**Pull request by Nielsien**
Added spatial axes measurement type and added a test for it in `device-measurement.test.ts`.

### CODE REVIEW

File: src/schemas.ts [hunk 1]
```diff
@@ -2,77 +2,77 @@
 // The schema-validator will pre-load these
 // This file will need to be manually updated with all schemas that are ready for consumption
 //
-import { JSONSchema7 } from 'json-schema';
```
Comment (Qwerios) [inline src/schemas.ts:5]: Please don't change the quote style like this in just a few files file. JSON is in " and the TypeScript code uses single quotes for string.

**Review by Qwerios — CHANGES_REQUESTED** [review]
The schema, changes and tests look fine to me. Could you please revert the changes to the quote style back to what it is in the rest of the codebase please.

File: src/schemas.ts (hunk 1 above)
Comment (Nielsien) [reply src/schemas.ts:5]: Hmm, pretty weird bug. My default prettier/ vs code config is set to single quotes but it just loves to use doubles for some reason. Anyway resolved the single quotes format; going to try reinstalling the extentions I have to solve this.

**Review by Qwerios — APPROVED** [review]

**Comment by Qwerios** [comment]
Published v0.0.9

**Pull request merged**